index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles.css" />
+    <title>NotesHD | Hetav Desai</title>
+  </head>
+  <body>
+    <script src="theme.js" type="text/javascript"></script>
+    <div class="container">
+      <section class="left">
+        <div class="header">
+          <h2 class="notes-list-heading">Notes HD</h2>
+          <div class="theme-switch-wrapper">
+            <label class="theme-switch" for="checkbox">
```

Comment:
`for` is not required when the input is itself nested inside the `label`. 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

Reply:
Also, since there is no text inside this label, there is basically no label for this chechbox. If someone uses a screen reader to use your app, they wont hear any description for this checkbox...they should hear something like `toggle theme`.

read more https://www.w3.org/WAI/tutorials/forms/labels/